src/main/java/org/jabref/gui/importer/actions/MergeReviewIntoCommentConfirmationDialog.java

```diff
@@ -25,13 +25,14 @@ public boolean askUserForMerge(List<BibEntry> conflicts) {
                 .map(Optional::get)
                 .collect(Collectors.joining(",\n"));
 
-        int answer = JOptionPane.showConfirmDialog(
+        String[] options = {Localization.lang("Merge fields")};
+        int answer = JOptionPane.showOptionDialog(
                 panel,
                 bibKeys + " " +
                         Localization.lang("has/have both a 'Comment' and a 'Review' field.") + "\n" +
                         Localization.lang("Since the 'Review' field was deprecated in JabRef 4.2, these two fields are about to be merged into the 'Comment' field.") + "\n" +
                         Localization.lang("The conflicting fields of these entries will be merged into the 'Comment' field."),
-                Localization.lang("Review Field Migration"), JOptionPane.DEFAULT_OPTION, JOptionPane.WARNING_MESSAGE);
+                Localization.lang("Review Field Migration"), JOptionPane.DEFAULT_OPTION, JOptionPane.WARNING_MESSAGE, null, options, options[0]);
```

Comment:
Maybe not a "Warning message", in fact it's an information message.  The user can't do anything about it.

Reply:
Well, infact it's a bit more important than just a simple notification, but I'd be open if others agree with you I'ts gonna be changed.